sdk/servicebus/service-bus/test/perf/service-bus-v1/receive.ts

```diff
@@ -8,47 +8,62 @@ Measures the maximum throughput of `receiver.receive()` in package `@azure/servi
 3. Set env vars `SERVICEBUS_CONNECTION_STRING` and `SERVICE_BUS_QUEUE_NAME`.
 4. This test presumes that there are messages in the queue.
 4. `ts-node receive.ts [totalMessages]`
```

Comment:
Is this...
```suggestion
4. `ts-node receive.ts [macConcurrentCalls] [totalMessages] [isReceiveAndDelete]`
```

Reply:
Fixed the instructions in other tests too.